ui-v2/app/templates/components/resolver-card.hbs

```diff
@@ -0,0 +1,46 @@
+<div class="resolver-card">
+  <header onclick={{onclick}} id={{concat 'resolver:' item.ID}}>
+    <a name="">
```

Comment:
If the anchor has no name or href, what's the value in using an anchor? To preserve the hover/click semantics?

Reply:
Hmm, yeah it's weird I remember playing with taking these `name=""`s out, let me take another look, I can't remember what the thinking was. I know that I was thinking 'i can see that these nodes are eventually gonna link somewhere to give users more info about the service' so thats why they are `a`s. Right now they act more like checkboxes rather than `a`'s or `buttons`'s, so semantically as they stand right now label/checkboxes might fit, but again I'm thinking they will eventually link somewhere.

I won't be making any major changes here for this I don't think, unless you feel super strongly, we can iterate on the accessibility aspect of it in another iteration, potentially before 1.7.0 final release.